prime-router/src/main/java/fhirengine/translation/hl7/structures/fhirinventory/segment/ORC.java

```diff
@@ -0,0 +1,1936 @@
+ * If you do not delete the provisions above, a recipient may use your version of
+ * this file under either the MPL or the GPL.
+ *
+ */
+
+
+package fhirengine.translation.hl7.structures.fhirinventory.segment;
+
+// import ca.uhn.hl7v2.model.v27.group.*;
+
+import ca.uhn.hl7v2.model.v27.datatype.*;
+import ca.uhn.hl7v2.HL7Exception;
+import ca.uhn.hl7v2.parser.ModelClassFactory;
+import ca.uhn.hl7v2.parser.DefaultModelClassFactory;
+import ca.uhn.hl7v2.model.AbstractMessage;
+import ca.uhn.hl7v2.model.Group;
+import ca.uhn.hl7v2.model.Type;
+import ca.uhn.hl7v2.model.AbstractSegment;
+import ca.uhn.hl7v2.model.Varies;
+
+
+/**
+ * <p>Represents an HL7 ORC message segment (Common Order).
+ * This segment has the following fields:</p>
+ * <ul>
+ * <li>ORC-1: Order Control (ID) <b> </b>
+ * <li>ORC-2: Placer Order Number (EI) <b>optional </b>
+ * <li>ORC-3: Filler Order Number (EI) <b>optional </b>
+ * <li>ORC-4: Placer Group Number (EI) <b>optional </b>
+ * <li>ORC-5: Order Status (ID) <b>optional </b>
+ * <li>ORC-6: Response Flag (ID) <b>optional </b>
+ * <li>ORC-7: Quantity/Timing (NULLDT) <b>optional repeating</b>
+ * <li>ORC-8: Parent (EIP) <b>optional </b>
+ * <li>ORC-9: Date/Time of Transaction (DTM) <b>optional </b>
+ * <li>ORC-10: Entered By (XCN) <b>optional repeating</b>
+ * <li>ORC-11: Verified By (XCN) <b>optional repeating</b>
+ * <li>ORC-12: Ordering Provider (XCN) <b>optional repeating</b>
+ * <li>ORC-13: Enterer's Location (PL) <b>optional </b>
+ * <li>ORC-14: Call Back Phone Number (XTN) <b>optional repeating</b>
+ * <li>ORC-15: Order Effective Date/Time (DTM) <b>optional </b>
+ * <li>ORC-16: Order Control Code Reason (CWE) <b>optional </b>
+ * <li>ORC-17: Entering Organization (CWE) <b>optional </b>
+ * <li>ORC-18: Entering Device (CWE) <b>optional </b>
+ * <li>ORC-19: Action By (XCN) <b>optional repeating</b>
+ * <li>ORC-20: Advanced Beneficiary Notice Code (CWE) <b>optional </b>
+ * <li>ORC-21: Ordering Facility Name (XON) <b>optional repeating</b>
+ * <li>ORC-22: Ordering Facility Address (XAD) <b>optional repeating</b>
+ * <li>ORC-23: Ordering Facility Phone Number (XTN) <b>optional repeating</b>
+ * <li>ORC-24: Ordering Provider Address (XAD) <b>optional repeating</b>
+ * <li>ORC-25: Order Status Modifier (CWE) <b>optional </b>
+ * <li>ORC-26: Advanced Beneficiary Notice Override Reason (CWE) <b>optional </b>
+ * <li>ORC-27: Filler's Expected Availability Date/Time (DTM) <b>optional </b>
+ * <li>ORC-28: Confidentiality Code (CWE) <b>optional </b>
+ * <li>ORC-29: Order Type (CWE) <b>optional </b>
+ * <li>ORC-30: Enterer Authorization Mode (CNE) <b>optional </b>
+ * <li>ORC-31: Parent Universal Service Identifier (CWE) <b>optional </b>
+ * <li>ORC-32: Advanced Beneficiary Notice Date (DT) <b>optional </b>
+ * <li>ORC-33: Alternate Placer Order Number (CX) <b>optional repeating</b>
+ * <li>ORC-34: Order Workflow Profile (EI) <b>optional repeating</b>
```

Comment:
The FHIR mapping inventory specifies this field as CWE but by all other sources EI is the correct type.

Reply:
Can we add this as a comment somewhere in the docs or in this file?